apigee.properties

```diff
@@ -2,11 +2,17 @@ category=External Analyzers
 description=Adds XML rules test Apigee apiproxies.
 homepageUrl=https://github.com/CreditMutuelArkea/sonar-apigee-plugin
 archivedVersions=1.1.0,1.2.0,1.3.0
-publicVersions=2.0.0
+publicVersions=2.1.0
 
 defaults.mavenGroupId=com.arkea.satd.sonar.apigee
 defaults.mavenArtifactId=sonar-apigee-plugin
 
+2.1.0.description=New features
+2.1.0.sqVersions=[6.7,LATEST]
+2.1.0.date=2020-04-08
+2.1.0.changelogUrl=https://github.com/CreditMutuelArkea/sonar-apigee-plugin/releases/tag/v2.1.0
+2.1.0.downloadUrl=https://github.com/CreditMutuelArkea/sonar-apigee-plugin/releases/download/v2.1.0/sonar-apigee-plugin-2.1.0.jar
+
 2.0.0.description=Support SonarXML 2.0.1+
 2.0.0.sqVersions=[6.7,LATEST]
```

Comment:
You can't have two versions compatible with `LATEST`. You need to set the end range for the one you're archiving to the current version of SQ: 8.2 in this case.

Reply:
Done.